configure

```diff
@@ -2695,17 +2695,19 @@ print_config "Windows PDB generation" "$pdb"
 ##########################################
 # check for timerfd support
 timerfd_create="no"
-cat > $TMPC << EOF
```

Comment:
Looking good! One minor suggestion - can you change the prefix from `bug:` to `configure:` as fio commit messages are prefixed by area rather than change type.